src/overview/components/Filters.jsx

```diff
@@ -3,7 +3,11 @@ import PropTypes from 'prop-types'
 
 import localStyles from './Filters.css'
 
-const Filters = ({ showOnlyBookmarks, onShowOnlyBookmarksChange }) => (
+const Filters = ({
+    showOnlyBookmarks,
+    onShowOnlyBookmarksChange,
+    onShowFilterChange,
```

Comment:
This callback, `onShowFilterChange`, is never being bound to anything in the `Filters` view. No need to pass it down here or the `Overview` view.

Reply:
My bad!